documentation/docs/guide/GettingStarted/deploy_locally.md

```diff
@@ -125,6 +125,8 @@ Once this is done you can use the following commands from any directory
 - `wp_log` - Tails the debug.log in the content directory
 - `gowp` - Goes to plugin directory
 - `gowp themes` - Goes to themes directory
+- `wp_setup_tests` - Sets up WordPress unit testing environment.
```

Comment:
i think these main points should be added.
* the WordPress instance needs to be up and running
* this only needs to be run once

Reply:
I feel like this is better explained in the section below already (https://github.com/bcgov/wordpress/pull/71/files#diff-5e7aeaa8a66429eeca33cffb5513ffc009e8849e3707a0064ad1d91a2b84307eR160).

Reply:
yes it has some additional information, but i don't think it addresses my points.

Reply:
even if this is the case, it might be helpful for this line to say something like... see the readme for more information, as this is not always implied

Reply:
I added a line about how the docker instance needs to be running to use the testing commands.
It's not 100% true that wp_setup_tests only needs to be run once I think. At least when I was running it locally it sometimes wiped the test files that the script downloads through svn because it stores those test files in a tmp directory.